crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1305,6 +1335,8 @@ pub struct ExtractMeshesSet;
 /// [`MeshUniform`] building.
 pub fn extract_meshes_for_cpu_building(
     mut render_mesh_instances: ResMut<RenderMeshInstances>,
+    mesh_material_ids: Res<RenderMaterialInstances>,
```

Comment:
Note: these appear to be correctly ordered via the constraints on `late_sweep_material_instances`.